audit/projects/k8s-infra-prow-build-trusted/secrets/cncf-ci-github-token/versions.json

```diff
@@ -1,4 +1,31 @@
 [
+  {
+    "createTime": "2021-06-15T19:22:05.290184Z",
+    "etag": "\"15c4d2e3a9dec8\"",
+    "name": "projects/180382678033/secrets/cncf-ci-github-token/versions/4",
```

Comment:
Expected, this was me flipping this secret to some dummy values and back to exercise kubernetes-external-secrets after https://github.com/kubernetes/k8s.io/pull/2219 had been deployed